#NAV_MSGS
Header header
float64 WGS84_Lat
float64 WGS84_Lon
float64 WGS84_Alt
float64 NED_N
float64 NED_E
float64 NED_D
float64 VEL_N
float64 VEL_E
float64 VEL_D
float64 ROLL
float64 PITCH
float64 YAW
float32 Uncertain_ROLL
float32 Uncertain_PITCH
float32 Uncertain_YAW
float32 Hori_Accuracy
float32 Vert_Accuracy

